src/apps/media/src/app/components/RenameFolderDialog.tsx

```diff
@@ -57,7 +57,7 @@ export const RenameFolderDialog = ({
 
   return (
     <Dialog open={open} onClose={onClose} fullWidth maxWidth={"xs"}>
-      <DialogTitle>Rename Folder</DialogTitle>
+      <DialogTitle fontWeight={700}>Rename Folder</DialogTitle>
```

Comment:
Why is `fontWeight` added to the `DialogTitle` component in this scenario but in the `NewFolderDialog` component an additional `Typography` component was included?

Reply:
My mistake on that one, updating it now

Reply:
updated the code now to use dialog title instead and made a PR in material repo for updating the font weight on theme level. Thank you
https://github.com/zesty-io/material/pull/88